modules/gdscript/gdscript_parser.cpp

```diff
@@ -944,11 +946,22 @@ void GDScriptParser::parse_class_member(T *(GDScriptParser::*p_parse_function)(b
 
 	// Consume annotations.
 	List<AnnotationNode *> annotations;
+#ifdef TOOLS_ENABLED
+	constexpr bool parsing_function = std::is_same_v<T, FunctionNode>;
+	AnnotationNode *îf_features = nullptr;
```

Comment:
Something is wrong with the variable name, here!

Reply:
Could you elaborate?

Reply:
There's a circumflex diacritic (`^`) on the `i`.

Reply:
Astigmatism? Presbyopia? Who knows... 😅

I thought using such characters in identifiers was still not possible, by the way.